app/src/main/java/com/example/simbirsoft_android_practice/help/HelpFragment.kt

```diff
@@ -1,48 +1,121 @@
 package com.example.simbirsoft_android_practice.help
 
+import android.content.Context
 import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
 import android.view.View
+import androidx.core.os.BundleCompat
+import androidx.core.view.isVisible
 import androidx.fragment.app.Fragment
 import androidx.recyclerview.widget.GridLayoutManager
 import com.example.simbirsoft_android_practice.R
 import com.example.simbirsoft_android_practice.core.CategoryRepository
 import com.example.simbirsoft_android_practice.core.JsonAssetExtractor
+import com.example.simbirsoft_android_practice.data.HelpCategory
 import com.example.simbirsoft_android_practice.databinding.FragmentHelpBinding
 import com.example.simbirsoft_android_practice.filter.CategoryMapper
 import dev.androidbroadcast.vbpd.viewBinding
+import java.util.concurrent.Executors
 
 private const val RECYCLER_VIEW_SPAN_COUNT = 2
+private const val KEY_CATEGORIES = "key_categories"
+private const val TIMEOUT = 5000L
```

Comment:
Уточняй единицы всегда
```suggestion
private const val TIMEOUT_IN_MILLIS = 5_000L
```